hub/powertoys/file-explorer.md

```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 
 ## Enable PowerToys Thumbnail Previews
 
+> [!NOTE]
+> A reboot might be required after enabling the thumbnail previewer for the settings to take effect.
+> [!WARNING]
+> Thumbnails might not appear on paths managed by cloud storage solutions like OneDrive, since these solutions may get their thumbnails from the cloud instead of generating them locally.
+
```

Comment:
```suggestion
> [!NOTE]
> A reboot might be required after enabling the thumbnail previewer for the settings to take effect.
> Thumbnails might not appear on paths managed by cloud storage solutions like OneDrive, since these solutions may get their thumbnails from the cloud instead of generating them locally.
```

Reply:
with my tweak and @Jay-o-Way  i like it.  i was about to comment to merge the two